src/Altinn.Platform/Altinn.Platform.Storage/Storage/Controllers/InstancesController.cs

```diff
@@ -277,7 +277,7 @@ public async Task<ActionResult<Instance>> Post(string appId, [FromBody] Instance
 
                 Instance instanceToCreate = CreateInstanceFromTemplate(appInfo, instance, creationTime, userId);
                 storedInstance = await _instanceRepository.Create(instanceToCreate);
-                await DispatchEvent(InstanceEventType.Created.ToString(), storedInstance);
+                await DispatchEvent(InstanceEventType.Created, storedInstance);
```

Comment:
Won't this store the int value rather than the string?

Reply:
There is still a ToString() in DispatchEvent()